pages/common/brew.md

```diff
@@ -0,0 +1,32 @@
+# brew
+
+> Package manager for macOS (or Linux).
```

Comment:
What about this?
```suggestion
> Package manager for macOS, Linux or Windows 10 Subsystem for Linux
```

Reply:
It also works in BSD derivations and many other systems, and WSL is just Linux in an emulator. Stating those 3 doesn't make much sense.

Reply:
OK, so does it make sense to merge in a common pages (I don't think there are pages for "macOS and Linux"). :innocent:

Reply:
If so, what about `Package manager for macOS and Linux` ?

Reply:
I think that's good, @nicokosi 👍🏻

Reply:
What about `Package manager for *nix systems.` ?

Reply:
That seems fine to me, except we need to be careful and make sure that we aren't being inaccurate at all (the Homebrew docs say macOS and Linux, they don't indicate anything else although it may work fine on other platforms)